File: src/main/java/cn/qiuye/gtmoremachine/data/lang/ItemLang.java
package cn.qiuye.gtmoremachine.data.lang;

import com.tterrag.registrate.providers.RegistrateLangProvider;

public class ItemLang {

    public static void init(RegistrateLangProvider provider) {
        provider.add("item.gtmoremachine.wireless_energy_receive_cover.tooltip.1", "§bPull Energy§7 from EU network to the machine as §fCover§7.");
        provider.add("item.gtmoremachine.wireless_energy_receive_cover.tooltip.2", "§7Can only used for §esingle block machine§7.Can't put on the machine blow the cover's voltage");
        provider.add("item.gtmoremachine.wireless_energy_receive_cover.tooltip.3", "§bEnergy transfer speed: §f%s §7EU/t");
        provider.add("item.gtmoremachine.wireless_transfer.tooltip.1", "§7Bind to: §f%s (%s)");
        provider.add("item.gtmoremachine.wireless_transfer.tooltip.2", "§7Right click the container with shift to bind container.Right click the air with shift to unbind.");
        provider.add("item.gtmoremachine.wireless_transfer.tooltip.bind.1", "Success bind to: %s (%s)");
        provider.add("item.gtmoremachine.wireless_transfer.tooltip.bind.2", "Success unbind.");
    }
}

File: src/main/java/cn/qiuye/gtmoremachine/data/lang/ItemLang.kt
package cn.qiuye.gtmoremachine.data.lang

import cn.qiuye.gtmoremachine.data.lang.LangHandler.addCN

object ItemLang {

	fun programmablec() {
		addCN("item.gtmoremachine.programmable_cover", "可编程覆盖板")
		addCN("item.gtmoremachine.virtual_item_provider", "虚拟物品提供器")
		addCN("item.gtmoremachine.virtual_item_provider_cell", "虚拟物品提供器元件")
	}

	fun itme() {
		addCN("item.gtmoremachine.advanced_terminal", "§b高级终端")
	}

	fun init() {
		programmablec()
		itme()
	}
}
